ddtrace/internal/symbol_db/symbols.py

```diff
@@ -509,14 +505,24 @@ def __len__(self) -> int:
 
 
 def is_module_included(module: ModuleType) -> bool:
+    # Check if module name matches the include patterns
     if symdb_config._includes_re.match(module.__name__):
         return True
 
-    package = packages.module_to_package(module)
-    if package is None:
+    # Check if it is user code
+    module_origin = origin(module)
+    if module_origin is None:
         return False
 
-    return symdb_config._includes_re.match(package.name) is not None
+    if packages.is_user_code(module_origin):
+        return True
+
+    # Check if the package name matches the include patterns
+    package = packages.filename_to_package(module_origin)
+    if package is not None and symdb_config._includes_re.match(package.name):
+        return True
```

Comment:
Just to double check the logic: this overrides the default excludes from the list as well as the additional excludes from the config env var?

I think I remember that you were removing the `includes` from the `excludes` when reading the list from file as well, right?

Reply:
The third-party list is used by `packages.is_user_code` (there is where we add the `excludes` to the list, and then remove the `includes`). So if a module is user code (based on the third-party detection configuration) we symbolise it. If it's not user code, we still make use of the include pattern from the SymDB configuration. So in essence the logic here is a logic OR between the SymDB pattern matching and the third-party detection.

Reply:
Ok, got it, that sounds good.